packages/react/src/date-picker/DatePickerInput.tsx

```diff
@@ -1,10 +1,12 @@
-import { forwardRef, MouseEventHandler, RefObject } from 'react';
+import { forwardRef, MouseEventHandler, RefObject, useMemo } from 'react';
+import { format } from 'date-fns';
 import { Flex } from '../flex';
 import { CalendarIcon } from '../icon';
 import { TextInputProps, textInputStyles } from '../text-input';
 import { mapSpacing } from '../core';
 import { Button } from '../button';
 import { Field } from '../field';
+import { acceptedDateFormats, AcceptedDateFormats } from './utils';
```

Comment:
Good pickup. I don't think there's been much consistency in this project, and I'll look to add linting for it in future because it's useful to know it's a type

Reply:
Fixed